daemon/algod/server.go

```diff
@@ -229,6 +229,9 @@ func (s *Server) Initialize(cfg config.Local, phonebookAddresses []string, genes
 			NodeExporterPath:          cfg.NodeExporterPath,
 		})
 
+	var algodEnvironmentAlgodVersionCounter = metrics.MakeCounter(metrics.MetricName{Name: "algod_environment_algod_version", Description: "Version of the Algod binary"})
+	algodEnvironmentAlgodVersionCounter.Inc(map[string]string{"version": config.GetAlgorandVersionAndBuild()})
```

Comment:
I've updated with requested changes, so that gauge from local clone now prints like:

```
# TYPE algod_environment_algod_version gauge
algod_environment_algod_version{version="3.26.0",commit="",channel="",goarch="arm64",goos="darwin"} 1
```